x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/cases/use_case_view_navigation.test.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+import { act, renderHook } from '@testing-library/react-hooks';
+import { useKibana } from '../../../../common/lib/kibana';
+import { AppMockRenderer, createAppMockRenderer } from '../../test_utils';
+import { useCaseViewNavigation } from './use_case_view_navigation';
+
+jest.mock('../../../../common/lib/kibana');
+
+const useKibanaMock = useKibana as jest.Mocked<typeof useKibana>;
+
+describe('useCaseViewNavigation', () => {
+  let appMockRender: AppMockRenderer;
+  const navigateToApp = jest.fn();
+
+  beforeEach(() => {
+    appMockRender = createAppMockRenderer();
+    useKibanaMock().services.application.currentAppId$ = new BehaviorSubject<string>('testAppId');
+    useKibanaMock().services.application.navigateToApp = navigateToApp;
+  });
+
+  it('calls navigateToApp with correct arguments', () => {
+    const { result, waitFor } = renderHook(() => useCaseViewNavigation(), {
+      wrapper: appMockRender.AppWrapper,
+    });
+
+    act(() => {
```

Comment:
do you really need to call act here? I tried without it I there is no warning or error message

Reply:
The examples in the [documentation](https://react-hooks-testing-library.com/usage/basic-hooks) of the library use `act` when you call a function. In react testing library it does not need it but I am not sure about the react testing hook library.